frame/merkle-mountain-range/src/tests.rs

```diff
@@ -83,6 +83,20 @@ fn add_blocks(blocks: usize) {
 	}
 }
 
+fn leaf_indices_to_block_numbers(
+	ext: &mut sp_io::TestExternalities,
+	leaf_indices: &Vec<LeafIndex>,
+) -> Vec<u64> {
+	leaf_indices
+		.iter()
+		.map(|l| {
+			let mmr_size = ext.execute_with(|| crate::Pallet::<Test>::mmr_leaves());
+			ext.execute_with(|| crate::Pallet::<Test>::leaf_index_to_parent_block_num(*l, mmr_size))
```

Comment:
Can we execute everything in a single `ext.execute_with()` block ? Do we need 2 blocks ?

Reply:
No, we don't :)